### PR TITLE
Add unit 5

### PR DESCRIPTION
**Pull request by dylanebert**

### CODE REVIEW

File: units/en/conclusion/conclusion.mdx [hunk 1]
```diff
@@ -1,5 +1,14 @@
 # Conclusion
 
-This course is in-progress and has not yet concluded!
+Thank you for participating in the Machine Learning for 3D Course!
 
-To receive updates as the course releases, sign up for the course mailing list [here](https://mailchi.mp/911880bcff7d/ml-for-3d-course).
+This has been a high-level overview of what's going on at the intersection of Machine Learning and 3D. For further exploration:
+
+- Keep up with [3D Arena](https://huggingface.co/spaces/dylanebert/3d-arena) for the latest projects.
```
Comment (Vaibhavs10) [inline units/en/conclusion/conclusion.mdx]: Could be cool to add a Hub URL with tag tracking all the open models as well for example: https://huggingface.co/models?pipeline_tag=image-to-3d&sort=trending

File: units/en/conclusion/conclusion.mdx [hunk 2]
```diff
@@ -1,5 +1,14 @@
...
+- Explore open-source projects like [InstantMesh](https://github.com/TencentARC/InstantMesh).
+
+![3D Arena](https://huggingface.co/datasets/dylanebert/ml-for-3d-course/resolve/main/3d-arena.png)
+
+As the field evolves rapidly, it's easy to get overwhelmed. Stay tuned for more accessible tools and resources as these projects mature.
```
Comment (Vaibhavs10) [inline units/en/conclusion/conclusion.mdx:13]: Is there a Discord channel/ track where people can discuss the course, we can link that here, we did this with audio course and people still use it to discuss things.

File: units/en/unit5/introduction.mdx [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  - Ensure there is a [model card](https://huggingface.co/docs/hub/en/model-cards).
+  - Tag the model with the `image-to-3d` pipeline tag.
+  - Apply the correct license if applicable.
+  
+- **Space**: A space hosted under your username, e.g. `hf.co/spaces/{username}/{space_name}`.
+  - Ensure the space converts images (`.png`, `.jpg`) to 3D (`.glb`, `.obj`, `.ply`, `.splat`).
+  - Reference your model URL in the space README.
+
+When completed, fill out this [form](https://forms.gle/rQedXFktHPYeikrt6) to receive your certificate.
+
```
Comment (Vaibhavs10) [inline units/en/unit5/introduction.mdx:23]: If we can give an explicit reference example or two here so that people can try it out, note: a lot of people would do what you show here i.e. the minimum requirements so it's good to leave it just enough open-ended that there's some creativity but not too open-ended in the sense that they take the easiest way out.

Comment (Vaibhavs10) [reply units/en/unit5/introduction.mdx:23]: So a good example would be to make a pipeline with text to image to 3D or other things where people can sequentially stack a couple of different models/APIs together.

File: units/en/unit5/introduction.mdx [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+## What's expected
+
+This capstone project is very open-ended. You can:
+
+- Build something from scratch.
+- Train or fine-tune an existing open-source model.
+- Clone and existing open-source model and change the demo experience.
+- Directly clone an open-source model and demo.
+
+Check out [3D Arena](https://huggingface.co/spaces/dylanebert/3d-arena) for the latest image-to-3D demos to use as starting points.
```
Comment (Vaibhavs10) [inline units/en/unit5/introduction.mdx:33]: Same as above, some examples or just a list of ideas would be quite helpful.

File: units/en/unit5/run-in-notebook.mdx [hunk 5]
```diff
@@ -0,0 +1,156 @@
...
+
+Then, run the demo code. This is exactly the same as in the space `app.py`. To ensure your model is working as expected, replace both instances of `dylanebert/LGM-full` with your `{username}/{model_name}`. Then, run the code.
+
+```python
+import shlex
+import subprocess
+
+import gradio as gr
+import numpy as np
+import spaces
```
Comment (Vaibhavs10) [inline units/en/unit5/run-in-notebook.mdx]: Do we need this dependency? we can simplify this without the `spaces` package stuff (so that people focus on the space instead of black magic packages)

File: units/en/unit5/run-in-notebook.mdx [hunk 6]
```diff
@@ -0,0 +1,156 @@
...
+import torch
+from diffusers import DiffusionPipeline
+```
+
+### Install diff-gaussian-rasterization
+
+For the gaussian splatting step of LGM, we need to install a custom wheel. This is a workaround for the space to run on [ZeroGPU](https://huggingface.co/zero-gpu-explorers).
```
Comment (Vaibhavs10) [inline units/en/unit5/run-in-notebook.mdx:92]: We can just remove this bit since people won't be running on ZeroGPU, right? Or is that intended?

If intended then it'd be good to add a reason/ ref for it.

File: units/en/unit5/run-in-notebook.mdx [hunk 7]
```diff
@@ -0,0 +1,156 @@
...
+subprocess.run(
+    shlex.split(
+        "pip install https://huggingface.co/spaces/dylanebert/LGM-mini/resolve/main/wheel/diff_gaussian_rasterization-0.0.0-cp310-cp310-linux_x86_64.whl"
+    )
+)
+```
+
+### Construct the pipeline
+
+Construct the [LGM](https://huggingface.co/dylanebert/LGM-full) pipeline. Replace `dylanebert/LGM-full` with your `{username}/{model_name}`.
```
Comment (Vaibhavs10) [inline units/en/unit5/run-in-notebook.mdx:104]: It'd be good to add a line about what this pipeline does (even if it was mentioned in earlier units, acts as a good way to ground the reader/ attendee)

File: units/en/unit5/run-in-notebook.mdx [hunk 8]
```diff
@@ -0,0 +1,156 @@
...
+    torch_dtype=torch.float16,
+    trust_remote_code=True,
+).to("cuda")
+```
+
+### Define the run function
+
+Define the run function that takes an image and returns a ply file.
+
+1. Convert the image to a numpy array and normalize it to [0, 1].
```
Comment (Vaibhavs10) [inline units/en/unit5/run-in-notebook.mdx:119]: Noob question: Why do we normalise? (might be good to add an explanation)

File: units/en/unit5/run-in-notebook.mdx [hunk 9]
```diff
@@ -0,0 +1,156 @@
...
+    inputs="image",
+    outputs=gr.Model3D(),
+    examples=[
+        "https://huggingface.co/datasets/dylanebert/iso3d/resolve/main/jpg@512/a_cat_statue.jpg"
+    ],
+    cache_examples=True,
+    allow_duplication=True,
+)
+demo.queue().launch()
+```
```
Comment (Vaibhavs10) [inline units/en/unit5/run-in-notebook.mdx:154]: Could we add a screenshot of how the demo would look like?

File: units/en/unit5/run-locally.mdx [hunk 10]
```diff
@@ -0,0 +1,33 @@
+# Run locally
+
+[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://githubtocolab.com/dylanebert/ml-for-3d-course-notebooks/blob/main/capstone.ipynb)
+
+The instructions below are tested on an RTX 4090 on WSL2 Ubuntu 22.04. Instructions will differ and may not work, depending on your setup.
+
+1. Install `git`, `python 3.10`, and `cuda` if not already installed.
```
Comment (Vaibhavs10) [inline units/en/unit5/run-locally.mdx:7]: Curious q: does this work locally on mac?

Note: people can also run the space via docker locally: https://huggingface.co/docs/hub/en/spaces-run-with-docker

Would be good to add that as an example too (it's okay to skip it if you think it's too much complexity)

File: units/en/unit5/run-via-api.mdx [hunk 11]
```diff
@@ -0,0 +1,35 @@
...
+    outputs=gr.Model3D(),
+    examples=[
+        "https://huggingface.co/datasets/dylanebert/iso3d/resolve/main/jpg@512/a_cat_statue.jpg"
+    ],
+    allow_duplication=True,
+)
+demo.queue().launch()
+```
+
+This will work on CPU, but relies on the original LGM-tiny, instead of your custom model. However, is your focus is on UI/UX or downstream tasks, this may be acceptable.
```
Comment (Vaibhavs10) [inline units/en/unit5/run-via-api.mdx:35]: This is a bit counter intuitive, we use `gradio_client` to run the same space as another space. Maybe we can add a section above just showing the use of `gradio_client`? this way it'll help make it clear that they can use other models/ spaces as well.

File: units/en/unit5/walkthrough.mdx [hunk 12]
```diff
@@ -0,0 +1,49 @@
...
+
+Congratulations! You've created a demo and met the minimum requirements for this capstone project.
+
+## So how do I run it?
+
+The demo requires a GPU, so it won't work on free hardware. However, there are many free options:
+
+1. **Run in this notebook**: Validate the code quickly.
+2. **Run locally**: Clone your space and run it locally.
+3. **Community grant**: Building something cool? Apply for a community GPU grant in your space settings.
```
Comment (Vaibhavs10) [inline units/en/unit5/walkthrough.mdx:45]: I'd add a mention of your Hub username so that you can explicitly see/ filter the community grants if asked.

**Review by Vaibhavs10 — COMMENTED** [review]
Great job with this! I mostly left some nits (some of these might not be applicable as you might have covered it earlier).
Cheers!
